test/ServiceRegistryL2.js

```diff
@@ -0,0 +1,1167 @@
+/*global describe, context, beforeEach, it*/
+
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+describe("ServiceRegistryL2", function () {
```

Comment:
It is a copy and paste I assume. It would be great if it was very easy to see the difference to L1

Reply:
It is not really a copy and paste, there are some function calls related to agent services missing, I didn't find a better way ATM

Reply:
Ok, so at a min, let's add a very basic check to the CI: `diff L1file L2file` and then assert the output is what we expect (a hardcoded string). That's a bit hacky but adds a lot of transparency on what exactly the diff is between the two versions :)